manifests/sftd/load-balancer/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: sftd
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/issuer: letsencrypt-staging
+    wire.com/host: sftd.k8s.arianvp.me
```

Comment:
probably change that ;-)

Reply:
I'll remvove this 'experimental' code into a separate branch for now